tests/Functional/DataFixtures/ORM/Api/WishlistTest/wishlist.yaml

```diff
@@ -54,23 +54,24 @@ Sylius\Component\Core\Model\Product:
     product_{1..10}:
         code: 'product_<current()>_code'
         enabled: true
-Sylius\Component\Core\Model\ProductTranslation:
-    product_translation_en_{1..10}:
-        translatable: '@product_<current()>'
-        name: '<word()>'
-        description: '<sentence()>'
-        shortDescription: '<sentence()>'
-        slug: '<slug()><current()>'
-        locale: 'en'
+#TODO: Load translations
+#Sylius\Component\Core\Model\ProductTranslation:
+#    product_translation_en_{1..10}:
+#        translatable: '@product_<current()>'
+#        name: '<word()>'
+#        description: '<sentence()>'
+#        shortDescription: '<sentence()>'
+#        slug: '<slug()><current()>'
+#        locale: 'en'
```

Comment:
Put the `locale` key first, setting the translatable before the locale code is set will result in errors. Same case w/ all other translations